packages/client/hmi-client/src/services/provenance.ts

```diff
@@ -21,6 +23,14 @@ import { getBulkModels } from './model';
 //        so we are using the following limit to optimize things a bit
 const MAX_RELATED_ARTIFACT_COUNT = 5;
 
+// Allowed types to search by in provenance so far...
+const ALLOWED_PROVENANCE_TYPES = [
+	ProvenanceType.Concept,
+	ProvenanceType.Dataset,
+	ProvenanceType.Model,
+	ProvenanceType.Project,
+	ProvenanceType.Publication
+];
```

Comment:
it looks like so far only a subset of provenance types are allowed to be search types